media-picker/src/main/java/com/miguelgaeta/media_picker/MediaPicker.java

```diff
@@ -190,8 +190,8 @@ public static void startForImageCrop(final Provider provider, final Uri uri, int
                 ? options
                 : new UCrop.Options();
 
-            // Force JPEG format compression now while working out filename strategy.
-            cropOptions.setCompressionFormat(Bitmap.CompressFormat.JPEG);
+            // Force PNG format compression
+            cropOptions.setCompressionFormat(Bitmap.CompressFormat.PNG);
```

Comment:
I believe this can just be deleted - we don't need to force it and can instead rely on the provided `UCrop.Options` to specify the compression format.

As it stands this change is dangerous because it will force all of our app's avatars to be png which I don't think we want across the board - only for emoji.